acceptance/Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gem 'chef-acceptance', github: 'chef/chef-acceptance'
+gem 'berkshelf'
```

Comment:
Do we need to add all of these stuff into the Gemfile? I think we are launching `kitchen` via CLI so these doesn't necessarily need to be gem dependencies. We can assume that the runner of `chef-acceptance` has installed Chef DK.